thinc/loss.py

```diff
@@ -246,12 +296,12 @@ def get_grad(
     def get_loss(
         self, guesses: Sequence[Floats2d], truths: Sequence[IntsOrFloatsOrStrs]
     ) -> float:
-        return self._get_loss_from_grad(self.get_grad(guesses, truths))
-
-    def _get_loss_from_grad(self, grads: Sequence[Floats2d]) -> float:
+        err = "Cannot calculate SequenceCategoricalCrossentropy loss: guesses and truths must be same length"
+        if len(guesses) != len(truths):  # pragma: no cover
+            raise ValueError(err)
```

Comment:
The error string can be passed to `ValueError` directly, I think. Same in the `get_grad` method above.